app/src/components/create_user.rs

```diff
@@ -303,19 +303,25 @@ impl Component for CreateUserForm {
     }
 }
 
+pub const REQUIRED_USER_ATTRIBUTES: &[&str] = &["mail"];
```

Comment:
Let's inline the value of this, it's going to be just "mail", we don't need a constant.